bpdm-pool/src/main/kotlin/org/eclipse/tractusx/bpdm/pool/service/RequestValidationService.kt

```diff
@@ -78,13 +76,43 @@ class RequestValidationService(
                             null,
                             LegalEntityCreateError.LegalEntityDuplicateIdentifier,
                             request.index
-                        )
-
+                        ) +
+                        validateLegalEntityDuplicates(legalEntity,duplicateIdentifiers,request.index)
             validationErrors.map { Pair(request, it) }
 
         }.groupBy({ it.first }, { it.second })
     }
 
+    fun validateLegalEntityDuplicates(legalEntity: LegalEntityDto, duplicateIdentifiers: Set<LegalEntityIdentifierDto>, entityKey: String?) : Collection<ErrorInfo<LegalEntityCreateError>> {
+        val errorList = mutableListOf<ErrorInfo<LegalEntityCreateError>>()
+        duplicateIdentifiers.forEach { duplicate ->
+            if (legalEntity.identifiers.contains(duplicate)) {
```

Comment:
Same as `findDuplicateLegalEntityIdentifiers` this should only consider the fields type and value.